app/helpers/application_helper.rb

```diff
@@ -1,4 +1,21 @@
 module ApplicationHelper
+
+  def only_user_initials(initials, limit: 2)
+    letters = initials.to_s.downcase.strip.split("")
```

Comment:
I suspect this whole method can be a lot simpler. Probably this will work:

def …
  return initials if initials.nil? || initials.length <= 2
  initials[0] + initials[-1]
end

I haven’t tested that so I may be overlooking something, but I think that’s a simple way to grab the first letter and the last letter.